docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.photo-album-slider.scss

```diff
@@ -0,0 +1,22 @@
+// This styling is for the Photo Gallery Grid and Photo Gallery Slideshow
+.ptype-stanford-gallery {
+  .su-gallery-images article,
+  .slick__slide article {
+    position: relative;
+  }
+
+  .su-gallery-description {
+    margin-bottom: hb-calculate-rems(32px);
```

Comment:
Can we use `margin-bottom: hb-spacing-width('md');` here? That will allow Traditional theme to have the tighter editorial spacing in comparison to the Colorful theme.